liquidprompt

```diff
@@ -1133,6 +1134,31 @@ _lp_python_env_color() {
     lp_python_env_color="[${LP_COLOR_VIRTUALENV}${lp_python_env}${NO_COL}]"
 }
 
+_lp_ruby_env() {
+    (( LP_ENABLE_RUBY_VENV )) || return 2
+
+    local ret
+
+    if command -v rvm-prompt >/dev/null ; then
```

Comment:
I forgot before now, sorry. We need a `_lp_require_tool` section for this, to prevent running this function every prompt if the user does not have one of these tools since this feature is enabled by default. Except `_lp_require_tool` can't be used, as there are two valid Ruby providing commands.

Throw this section after line 497:
```bash
    if (( LP_ENABLE_RUBY_VENV )) \
      && ! command -v rvm-prompt \
      && ! command -v rbenv ; then
        LP_ENABLE_RUBY_VENV=0
    fi
````

Reply:
I just pushed a proposal in 72c01b1afb064814236bf29ca7c8aee8400d1cdb. What do you think of it?

Reply:
Yes, that looks good. The only issues are:

1. Delete line 506. Setting no value is pointless if the feature is disabled.
2. Use `[[` instead of `[` everywhere.